### PR TITLE
fix: try disabling parallel for this test

### PR DESCRIPTION
**Pull request by pmuir**
#### Submitter checklist

- [ ] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.

#### Which issue this PR fixes

fixes #

### CODE REVIEW

File: pkg/jx/cmd/upgrade_apps_test.go [hunk 1]
```diff
@@ -92,8 +92,8 @@ func TestUpgradeAppForGitOps(t *testing.T) {
 	assert.Equal(t, newVersion.String(), found[0].Version)
 }
 
-func TestUpgradeAppWithShortNameForGitOps(t *testing.T) {
-	t.Parallel()
+func qTestUpgradeAppWithShortNameForGitOps(t *testing.T) {
```
Comment (abayer) [inline pkg/jx/cmd/upgrade_apps_test.go]: Pretty sure you didn't mean to put a `q` there. =)

Comment (jstrachan) [reply pkg/jx/cmd/upgrade_apps_test.go]: removed it :)

**Pull request merged**